dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -7,9 +7,24 @@
  */
 
 #include "precomp.h"
+#include <compat_undoc.h>
 
 WINE_DEFAULT_DEBUG_CHANNEL(dmenu);
 
+static inline bool RegValueExists(HKEY hKey, LPCWSTR Name)
+{
+    return RegQueryValueExW(hKey, Name, NULL, NULL, NULL, NULL) == ERROR_SUCCESS;
+}
+
+static BOOL InsertMenuItemAt(HMENU hMenu, UINT Pos, UINT Flags)
+{
+    MENUITEMINFOW mii;
+    mii.cbSize = FIELD_OFFSET(MENUITEMINFOW, hbmpItem);
```

Comment:
What is the point of faking a smaller struct size?

Reply:
To disconnect shell32 from user32 and be compatible up and down forever.

Reply:
In that case I'd like to ask that you include that as comment.